itou/www/approvals_views/views.py

```diff
@@ -456,12 +462,27 @@ def get_context_data(self, **kwargs):
 class ProlongationRequestShowView(ProlongationRequestViewMixin, TemplateView):
     template_name = "approvals/prolongation_requests/show.html"
 
+    def get_context_data(self, **kwargs):
+        data = super().get_context_data(**kwargs)
+        data["block_validation"] = self.prolongation_request.start_at < max(
+            self.prolongation_request.approval.prolongation_set.values_list("end_at", flat=True)
+        )
```

Comment:
Je réfléchis à voix haute mais je me demande si faire ceci ne suffirais pas, c'est pas tout à fait pareil mais je me dit que ça pourrais aussi choper les erreurs liées aux suspensions :
```suggestion
        data["block_validation"] = self.prolongation_request.start_at != self.prolongation_request.approval.end_at
```